src/App.tsx

```diff
@@ -15,47 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type Goods = string[];
+
+const SORT_FIELD_BY_LETTERS = 'letters';
```

Comment:
The name of the button for this type is called "Sort alphabetically", so it is better to stick to the same style in the name.